### DISCUSSION TITLE
Welcome to winimerge Discussions!

### DISCUSSION THREAD

**Discussion by sdottaka** [WinMerge/winimerge#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Teknikaz-Unidaz** [comment]
Hello, we're a very small IT company that always uses your software to compare files (thank you so much for creating this powerful tool). We've signed up to support the community by providing feedback on usage issues with the software for further improvement.

Greetings to the creators and the entire community.
